diff --git a/src/Pyz/Yves/Router/RouterDependencyProvider.php b/src/Pyz/Yves/Router/RouterDependencyProvider.php
index e13a2a8bc..ec52b08e6 100644
--- a/src/Pyz/Yves/Router/RouterDependencyProvider.php
+++ b/src/Pyz/Yves/Router/RouterDependencyProvider.php
@@ -10,6 +10,7 @@ declare(strict_types = 1);
 namespace Pyz\Yves\Router;
 
 use Pyz\Yves\ExampleProductSalePage\Plugin\Router\ExampleProductSaleRouteProviderPlugin;
+use Pyz\Yves\HelloWorld\Plugin\Router\HelloWorldRouteProviderPlugin;
 use Pyz\Yves\MultiCartPage\Plugin\Router\MultiCartPageRouteProviderPlugin;
 use Spryker\Yves\CustomerDataChangeRequest\Plugin\Router\CustomerDataChangeRequestRouteProviderPlugin;
 use Spryker\Yves\HealthCheck\Plugin\Router\HealthCheckRouteProviderPlugin;
@@ -196,6 +197,7 @@ class RouterDependencyProvider extends SprykerRouterDependencyProvider
             new CostCenterRouteProviderPlugin(),
             new PunchoutGatewayRouteProviderPlugin(),
             new SecurityOauthKnpuRouteProviderPlugin(),
+            new HelloWorldRouteProviderPlugin(),
         ];
     }
